docs/development_suite/overview-dev-suite.md

```diff
@@ -4,28 +4,47 @@ title: Console Overview
+* **Lower cognitive load**: By abstracting underlying complexities, the tool simplifies tasks for developers, minimizing cognitive load and reducing the need for frequent context switching. It consolidates all necessary project information, from configuration to monitoring, into a single interface. This allows developers to focus more on creative problem-solving and efficient coding, ultimately enhancing productivity and innovation within development projects.
+
+## Ease the digital platform development
+
+Mia-Platform Console serves the whole team (Developer, Operations and Platform Engineering) involved in software development because it brings several advantages to the development of an Internal Developer Platform.
+
+- **Governance & controls**: Manage runtime resources across hybrid and multi-cloud architectures, connect clusters, establish self-service environments and enforce RBAC policies, by creating 'paved roads'. You can also establish project templates that standardize developer tools and the ability to choose and connect your own provider such as Git.
+Using a Software Catalog, each team owns one or a group of projects.
+
+* **Productivity & quality**: Software can be built in a unified environment that offers secure, self-service capabilities. By leveraging the GitOps methodology, applications can be exposed and deployed seamlessly from a centralized setup.
+Enhance your development speed with ready-to-use services or by directly uploading Docker images.
+
+* **Resilience & Observability**: Ensure long-term system integrity through advanced observability tools, metrics, and alerting mechanisms.
+Enhance services usability with a functional document portal and with API testing capabilities.
+
+The use of Mia-Platform Console does not bind in any way to a specific technology stack. This product integrates various technologies, ranging from front-end development to deployment tools.
+
+Leverage the **[Marketplace](/marketplace/overview_marketplace.md)** to accelerate development. Thanks to this self-service catalog you can find a set of curated, standardized, and reusable software components to easily integrate in your Internal Developer Portal (IDP). 
+
+## Hierarchical Organization of Mia-Platform Console
 
 Mia-Platform Console is organized into hierarchical levels to help users navigate its structure effectively. Starting from the highest level down to the most detailed, this section provides a quick guide for understanding the architecture.
```

Comment:
The Mia-Platform Console is structured hierarchically to facilitate efficient navigation for users. This section offers a concise guide to understanding its architecture, starting from the highest level down to the most detailed.